app/collections/settings.spec.js

```diff
@@ -19,48 +19,61 @@ const collectionWithoutDefaultSubject = {
   display_name: 'A collection',
   private: false,
   slug: 'username/a-collection'
-}
+};
 
-describe('<CollectionSettings />', function() {
+describe('<CollectionSettings />', () => {
```

Comment:
I haven't been a stickler about this for PFE tests, but the mocha docs specifically say that arrow functions are discouraged: https://mochajs.org/#arrow-functions

Changing this isn't necessary.

Reply:
Happy to not use arrow functions, as this is what Mocha prefers :) I was using arrow functions because function expressions are generating the linter errors and warning below for every test: 
```
    warning  Unexpected unnamed function                               func-names
    error    Unexpected function expression                            prefer-arrow-callback
    error    Missing space before function parentheses                 space-before-function-paren
```
should we modify our eslint config to not throw these? or just ignore them?

Reply:
At some point we can setup the linter to have different rules for mocha, but that's out of scope here. Just ignore them or you can put exceptions at the top of the file if it's bothersome.